pkg/keyspace/tso_keyspace_group.go

```diff
@@ -147,7 +161,8 @@ func (m *GroupManager) Bootstrap() error {
 		m.groups[userKind].Put(group)
 	}
 
-	return nil
+	// Load all the keyspaces from the storage and assign them to the respective keyspace groups.
+	return m.patrolKeyspaceAssignmentLocked()
```

Comment:
Will it affect the time of the transfer leader?

Reply:
It will, so I removed this patrol to make it only run before the first split. PTAL.